sw/device/lib/runtime/uart_stdout.c

```diff
@@ -0,0 +1,26 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "sw/device/lib/runtime/uart_stdout.h"
+
+#include "sw/device/lib/dif/dif_uart.h"
+#include "sw/device/lib/base/print.h"
+
+size_t uart_send_buf(void *data, const char *buf, size_t len) {
```

Comment:
Should it be `static` ?

Reply:
Yes. It becomes static in another commit, but I can fix it so that it's static in the commit that introduces it.